server/src/main/java/org/apache/iotdb/db/qp/physical/crud/AlignByDevicePlan.java

```diff
@@ -18,18 +18,21 @@
  */
 package org.apache.iotdb.db.qp.physical.crud;
 
-import java.util.List;
-import java.util.Map;
 import org.apache.iotdb.db.metadata.PartialPath;
 import org.apache.iotdb.db.qp.logical.Operator;
 import org.apache.iotdb.db.qp.logical.Operator.OperatorType;
 import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
 import org.apache.iotdb.tsfile.read.expression.IExpression;
 
+import java.util.List;
+import java.util.Map;
+
 public class AlignByDevicePlan extends QueryPlan {
 
-  private List<String> measurements; // to record result measurement columns, e.g. temperature, status, speed
-  private Map<String, String> measurementAliasMap; // select s1, s2 as speed from root, then s2 -> speed
+  private List<String>
+      measurements; // to record result measurement columns, e.g. temperature, status, speed
+  private Map<String, String>
```

Comment:
Don't use end-of-line comments?

Reply:
At present, we have a lot of end of line comments, which are very convenient. If we want to prohibit the use of end of line comments, we need to initiate a discussion. due to we 
 may  need  modify Google-Java-format. I suggest solving it in another PR after discussion. What do you think?

Reply:
okay